src/ray/gcs/redis_async_context.h

```diff
@@ -32,6 +36,12 @@ typedef void redisCallbackFn(struct redisAsyncContext *, void *, void *);
 
 namespace ray {
 namespace gcs {
+// C wrappers for class member functions
+extern "C" void call_C_addRead(void *private_data);
```

Comment:
May I ask what's the point of placing `extern "C"` into namespace?
`extern "C"` is made for exporting C-compatible symbols, in other words, unmangled.
Placing them into namespace will have them mangled again... Not sure what's your intention.

Code to check:
```
extern "C" {
void FuncOutOfNamespace() {}
}  // extern "C"
namespace ray {
void FuncInsideOfNamespace() {}
}  // namespace ray
```
Check ELF symbol:
```
[~] 
ubuntu@hjiang-devbox-pg$ g++ -std=c++17 -c extern_c_test.cc -o test

[~] 
ubuntu@hjiang-devbox-pg$ nm -C test
0000000000000000 T FuncOutOfNamespace
000000000000000b T ray::FuncInsideOfNamespace()
```

Reply:
The original file has extern "C", but I found that it seems that we don't need it. So I'll delete them.

Reply:
My rough guess is to provide c hook to redis library, but I don't think extern c is a necessity in general case.

Reply:
fixed